Formula/ghostunnel.rb

```diff
@@ -0,0 +1,24 @@
+class Ghostunnel < Formula
+  desc "Simple SSL/TLS proxy with mutual authentication"
+  homepage "https://github.com/ghostunnel/ghostunnel"
+  url "https://github.com/ghostunnel/ghostunnel/archive/refs/tags/v1.6.0.tar.gz"
+  sha256 "c86c04b927b45cbaa3dbb6339d9e89e00fe0f4235bb3c0a50c837f3f89a42df4"
+  license "Apache-2.0"
+  head "https://github.com/ghostunnel/ghostunnel.git", branch: "master"
```

Comment:
just some nitpick comment in here, we default head build for master branch. :)

Reply:
@chenrui333 We now require the default branch to be specified, see Homebrew/brew#11720.